datashuttle/utils/utils.py

```diff
@@ -9,11 +13,30 @@
 # --------------------------------------------------------------------------------------------------------------------
 
 
-def message_user(message: Union[str, list]) -> None:
+def log(message: str) -> None:
+    logger = logging.getLogger("fancylog_")
```

Comment:
You probably don't want the logger named `fancylog`. What if you have >1 package installed that are using fancylog?